From 1c7f9a676088ecee4c14df14b8688b391fb32a05 Mon Sep 17 00:00:00 2001
From: Justin Stephenson <jstephen@redhat.com>
Date: Mon, 19 Dec 2016 16:49:17 -0500
Subject: [PATCH 12/79] FAILOVER: Improve port status log messages

It should be more clear to administrators that when SSSD internal
port status is set as PORT_NOT_WORKING, this does not directly relate
to an assumed network port-related issue.

Reviewed-by: Jakub Hrozek <jhrozek@redhat.com>
---
 src/providers/fail_over.c | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/src/providers/fail_over.c b/src/providers/fail_over.c
index 5d3c26d4a690769637f2fa4f41a76627cbdba77a..168e59d6f4e9fc8abd827be21004daef2c6613f0 100644
--- a/src/providers/fail_over.c
+++ b/src/providers/fail_over.c
@@ -376,12 +376,18 @@ get_port_status(struct fo_server *server)
           "Port status of port %d for server '%s' is '%s'\n", server->port,
               SERVER_NAME(server), str_port_status(server->port_status));
 
+    if (server->port_status == PORT_NOT_WORKING) {
+        DEBUG(SSSDBG_MINOR_FAILURE, "SSSD is unable to complete the full "
+              "connection request, this internal status does not necessarily "
+              "indicate network port issues.\n");
+    }
+
     timeout = server->service->ctx->opts->retry_timeout;
     if (timeout != 0 && server->port_status == PORT_NOT_WORKING) {
         gettimeofday(&tv, NULL);
         if (STATUS_DIFF(server, tv) > timeout) {
             DEBUG(SSSDBG_CONF_SETTINGS,
-                  "Reseting the status of port %d for server '%s'\n",
+                  "Resetting the status of port %d for server '%s'\n",
                       server->port, SERVER_NAME(server));
             server->port_status = PORT_NEUTRAL;
             server->last_status_change.tv_sec = tv.tv_sec;
-- 
2.9.3

